src/app/total/total.component.html

```diff
@@ -0,0 +1,8 @@
+<!-- <p>total works!</p> -->
+<div>
+    <h3>Total</h3>
+  </div>
+  
+  <div>
+    <span class="total-number">{{this.countTotal()}}</span>
```

Comment:
We could use the `currency` pipe here to give us currency details such as the symbol